docs/bokeh/source/docs/releases/3.1.0.rst

```diff
@@ -10,3 +10,4 @@ Changes
 
 * Official support for Python 3.11 was added
 * ``select_every_mousemove`` was deprecated and renamed to ``continuous``
+* Internal compilation target was upgraded to ES2022. This may affect thirdparty extensions and custom builds of bokehjs.
```

Comment:
This is a big vague, what do extension authors need to _do_? Seems like something to add to migration guide wiki?

Reply:
It's vague, because it depends on what extensions' authors have done, their tsconfigs etc. Most likely they will have to do nothing (e.g. panel doesn't use `override` yet, so this change doesn't affect it), or at worst they will have to do what I did in this PR. In any case, they will get feedback from the compiler, which will tell them what to do.

Reply:
That being said, some form of migration guide may be useful, even if just showing potentially needed changes.